packages/zapp/console/src/components/Literals/LiteralMapViewer.tsx

```diff
@@ -40,11 +50,25 @@ export const LiteralMapViewer: React.FC<{
       <NoneTypeValue />
     </div>
   );
+
+  let rootNode: string | null = null;
+
+  const transformedLiteralsKeys = Object.keys(transformedLiterals);
```

Comment:
nit: it makes sense to me that we only set the name of the root node when there is one key in the first layer of json. Personally, I think it will be good to make it to a function or leave some comments! overall, it looks good to me